### PR TITLE
{chem,phys}[GCC/10.2.0,foss/2020b] Elk v7.0.12, libxc v5.1.2

### PR DESCRIPTION
**Pull request by zarybnicky**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/e/Elk/Elk-7.0.12-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+files_to_copy = [(
+    [
+        'src/elk', 'src/spacegroup/spacegroup', 'src/eos/eos',
+        'utilities/elk-bands/elk-bands', 'utilities/elk-optics/elk-optics.py',
+        'utilities/wien2k-elk/se.pl'
+    ], 'bin'
+)]
+
+sanity_check_paths = {
+    'files': ['bin/elk', 'bin/spacegroup', 'bin/eos'],
```
Comment (boegel) [inline easybuild/easyconfigs/e/Elk/Elk-7.0.12-foss-2020b.eb:56]: @zarybnicky Any sanity check command we can add for `elk`?

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3508.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/657b60fb8aef9227f5ebbf54c7512545 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3125.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/4eda162434bf311fb090602f32765116 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node2602.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/01630c72704f660c77a4bea862dd8d4b for a full test report.

File: easybuild/easyconfigs/e/Elk/Elk-7.0.12-foss-2020b.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/e/Elk/Elk-7.0.12-foss-2020b.eb:56]: fixed in 5aab141c21

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3516.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/51d9225b7a07864038202e9c902760a6 for a full test report.

File: easybuild/easyconfigs/e/Elk/Elk-7.0.12-foss-2020b.eb (hunk 1 above)
Comment (zarybnicky) [reply easybuild/easyconfigs/e/Elk/Elk-7.0.12-foss-2020b.eb:56]: Nice, thanks, I was just in the process of waiting for libxc to build :)

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @zarybnicky!

**Pull request merged**